### PR TITLE
Default row height API (#41)

### PR DESCRIPTION
**Pull request by anayini**
## Changes in this pull request

Issue fixed: #41

### Checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I added an entry to the `CHANGELOG.md` for any breaking changes, enhancements, or bug fixes.
- [x] I have reviewed the [contributing guide](https://github.com/plangrid/ReactiveLists/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: Tests/TableView/TestTableViewModels.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ import Foundation
 @testable import ReactiveLists
 
 struct TestCellViewModel: TableCellViewModel {
-    let rowHeight: CGFloat = 42
+    let rowHeight: CGFloat? = 42
```
Comment (anayini) [inline Tests/TableView/TestTableViewModels.swift]: The bug was overriding a default variable but we forgot this `?` so it was a new variable.

File: Sources/TableViewModel.swift [hunk 2]
```diff
@@ -58,9 +58,9 @@ public protocol TableCellViewModel: ReusableCellViewModelProtocol, DiffableViewM
 /// Default implementations for `TableCellViewModel`.
 public extension TableCellViewModel {
 
-    /// Default implementation, returns `44.0`.
-    var rowHeight: CGFloat {
-        return 44.0
+    /// Default implementation, returns `nil`.
```
Comment (jessesquires) [inline Sources/TableViewModel.swift]: how about:

```
/// Default implementation, returns `nil`. 
/// - Note: If `nil`, the `TableViewDriver` will fallback to `TableViewModel.defaultRowHeight`.
/// - Seealso: TableViewModel
```

Comment (anayini) [reply Sources/TableViewModel.swift]: 👍

**Comment by jessesquires** [comment]
ah gotcha 👍 LGTM then

**Pull request merged**